tests/ops/api/v1/endpoints/test_masking_endpoints.py

```diff
@@ -7,33 +7,31 @@
 from fidesops.ops.schemas.masking.masking_configuration import (
     AesEncryptionMaskingConfiguration,
 )
+from fidesops.ops.service.masking.strategy.masking_strategy import MaskingStrategy
 from fidesops.ops.service.masking.strategy.masking_strategy_aes_encrypt import (
-    AES_ENCRYPT_STRATEGY_NAME,
-)
-from fidesops.ops.service.masking.strategy.masking_strategy_factory import (
-    MaskingStrategyFactory,
+    AesEncryptionMaskingStrategy,
 )
 from fidesops.ops.service.masking.strategy.masking_strategy_hash import (
-    HASH_STRATEGY_NAME,
+    HashMaskingStrategy,
 )
 from fidesops.ops.service.masking.strategy.masking_strategy_hmac import (
-    HMAC_STRATEGY_NAME,
+    HmacMaskingStrategy,
 )
 from fidesops.ops.service.masking.strategy.masking_strategy_nullify import (
-    NULL_REWRITE_STRATEGY_NAME,
+    NullMaskingStrategy,
 )
 from fidesops.ops.service.masking.strategy.masking_strategy_random_string_rewrite import (
-    RANDOM_STRING_REWRITE_STRATEGY_NAME,
+    RandomStringRewriteMaskingStrategy,
 )
 from fidesops.ops.service.masking.strategy.masking_strategy_string_rewrite import (
-    STRING_REWRITE_STRATEGY_NAME,
+    StringRewriteMaskingStrategy,
 )
 
 
 class TestGetMaskingStrategies:
     def test_read_strategies(self, api_client: TestClient):
         expected_response = []
-        for strategy in MaskingStrategyFactory.get_strategies():
+        for strategy in MaskingStrategy.get_strategies():
             expected_response.append(strategy.get_description())
 
         response = api_client.get(V1_URL_PREFIX + MASKING_STRATEGY)
```

Comment:
yeah, i've been a bit uncertain about this test since i first saw it. but i'm also not really sure on the best way to resolve it - i feel like any approach to getting a "true" list of strategies is either going to rely on duplicating the logic of `get_strategies()`, or it's going to be prone to false negatives if/when more masking strategies are added to the testing runtime (whether that's core fidesops updates or, potentially, some extended runtime like `-plus`).

obviously we need more robust testing for `get_strategies()`, as you've correctly pointed out. but what do you think about keeping this as is, and just focusing on firming up the tests around `get_strategies()` itself?

Reply:
The get_strategies() improvement was the main thing, but perhaps asserting that the response is non-empty at least?